packages/core-data/src/resolvers.js

```diff
@@ -801,6 +801,26 @@ export const getRevisions =
 			false,
 			meta
 		);
+
+		// When requesting all fields, the list of results can be used to
+		// resolve the `getEntityRecord` selector in addition to `getEntityRecords`.
```

Comment:
Looks like a copy / paste 😄

```suggestion
		// resolve the `getRevision` selector in addition to `getRevisions`.
```